proto/Networking/Requests/RequestType.proto

```diff
@@ -107,77 +107,94 @@ enum RequestType {
 	OPT_OUT_PUSH_NOTIFICATION_CATEGORY = 5003;  // Implemented
 	DOWNLOAD_GAME_MASTER_TEMPLATES = 5004;      // Implemented
 	GET_INVENTORY = 5005;                       // UnUsed - Implemented
 	REDEEM_PASSCODE = 5006;                     // Implemented
-	PING = 5007;                                // 
-	ADD_LOGIN_ACTION = 5008;                    // 
-	REMOVE_LOGIN_ACTION = 5009;                 // 
+	PING = 5007;                                // Implemented
+	ADD_LOGIN_ACTION = 5008;                    // Implemented
+	REMOVE_LOGIN_ACTION = 5009;                 // Implemented
 	LIST_LOGIN_ACTION = 5010;                   // Implemented?
 	ADD_NEW_POI = 5011;                         // Implemented
-	PROXY_SOCIAL_ACTION = 5012;                 // 
-	CLIENT_TELEMETRY = 5013;                    // 
+	PROXY_SOCIAL_ACTION = 5012;                 // ???
+	CLIENT_TELEMETRY = 5013;                    // Implemented
 	GET_AVAILABLE_SUBMISSIONS = 5014;           // Implemented
-	GET_SIGNED_URL_FOR_PHOTO_UPLOAD = 5015;
-	REPLACE_LOGIN_ACTION = 5016;
-	PROXY_SOCIAL_SIDE_CHANNEL_ACTION = 5017;
-	COLLECT_CLIENT_TELEMETRY = 5018;
-	PURCHASE_SKU = 5019;
-	GET_AVAILABLE_SKUS_AND_BALANCES = 5020;
-	REDEEM_GOOGLE_RECEIPT = 5021;
-	REDEEM_APPLE_RECEIPT = 5022;
-	REDEEM_DESKTOP_RECEIPT = 5023;
-	UPDATE_FITNESS_METRICS = 5024;
-	GET_FITNESS_REPORT = 5025;					// Implemented
-	GET_CLIENT_TELEMETRY_SETTINGS = 5026;
-	REGISTER_BACKGROUND_SERVICE = 5028;
-	GET_CLIENT_BGMODE_SETTINGS = 5029;
-	PING_ASYNC = 5027;
-	PING_DOWNSTREAM = 5030;
-	SET_IN_GAME_CURRENCY_EXCHANGE_RATE = 5032;	// Implemented
-	REQUEST_GEOFENCE_UPDATES = 5033;			// Implemented
-	UPDATE_PLAYER_LOCATION = 5034;				// Implemented
-	PROFANITY_FILTER_ACTION = 5035;
+	GET_SIGNED_URL_FOR_PHOTO_UPLOAD = 5015;     // Implemented
+	REPLACE_LOGIN_ACTION = 5016;                // Implemented
+	PROXY_SOCIAL_SIDE_CHANNEL_ACTION = 5017;    // Implemented
+	COLLECT_CLIENT_TELEMETRY = 5018;            // Implemented
+	PURCHASE_SKU = 5019;                        // ???
+	GET_AVAILABLE_SKUS_AND_BALANCES = 5020;     // ???
+	REDEEM_GOOGLE_RECEIPT = 5021;               // ???
+	REDEEM_APPLE_RECEIPT = 5022;                // ???
+	REDEEM_DESKTOP_RECEIPT = 5023;              // ???
+	UPDATE_FITNESS_METRICS = 5024;              // Implemented
+	GET_FITNESS_REPORT = 5025;                  // Implemented
+	GET_CLIENT_TELEMETRY_SETTINGS = 5026;       // Implemented
+	REGISTER_BACKGROUND_SERVICE = 5028;         // ???
+	GET_CLIENT_BGMODE_SETTINGS = 5029;          // Implemented
+	PING_ASYNC = 5027;                          // ???
+	PING_DOWNSTREAM = 5030;                     // ???
+	SET_IN_GAME_CURRENCY_EXCHANGE_RATE = 5032;  // Implemented
+	REQUEST_GEOFENCE_UPDATES = 5033;            // Implemented
+	UPDATE_PLAYER_LOCATION = 5034;              // Implemented
+	PROFANITY_FILTER_ACTION = 5035;             // ???
+	// SocialActions
+	SEARCH_PLAYER = 10000;                 // Implemented
```

Comment:
no social actions are not supposed to be there

Reply:
ok faudrais enlever // social..  comment j'ai mis pour me orienter voir se que est ok ou pas faudra le enlever 👍

Reply:
faut enlever les actions pas que le commentaire
les actions social ne sont pas ici

Reply:
ça marche moi je t’écoute

Reply:
fixed https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/97be62a1368a48568334942b6d64679d681dd1a7